plugins/de.ovgu.featureide.fm.attributes/src/de/ovgu/featureide/fm/attributes/base/impl/StringFeatureAttribute.java

```diff
@@ -86,4 +86,9 @@ public IFeatureAttribute cloneAtt(IFeature feature) {
 	public IFeatureAttribute cloneRecursive(IFeature feature) {
 		return new StringFeatureAttribute(feature, this.getName(), this.getUnit(), null, this.isRecursive(), this.isConfigurable());
 	}
+
+	@Override
+	public boolean isValidValue(String value) {
+		return true;
```

Comment:
same for this

Reply:
Currently intended. We can discuss possible limitations though (e.g., regarding length).